plugins/field-date/test/field_date_test.mocha.js

```diff
@@ -117,7 +139,112 @@ suite('FieldDate', function() {
       });
       test('New Value', function() {
         this.field.setValue('3030-03-30');
-        assertFieldValue(this.field, '3030-03-30');
+        assertFieldValue(this.field, '3030-03-30', '3/30/3030');
+      });
+    });
+  });
+
+  suite('Time Zones', function() {
```

Comment:
Thank you for adding time zone tests :D

Reply:
Sure thing! They seemed important for conveying what I designed the timezone behavior around.